app/serializers/api/v1/active_model_errors_serializer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ActiveModelErrorsSerializer
+      include ActiveModel::Model
+      include ActiveModel::Attributes
+      include ActiveModel::Serialization
+
+      attribute :title
+      attribute :detail
+
+      def self.from(service)
+        {
+          errors: service
+            .errors
+            .messages
+            .map { |title, detail| new(title:, detail: detail.uniq.join(", ")).serializable_hash },
```

Comment:
I just wanted to confirm we are happy moving away from the error structure/titles we have now such as `"Bad or missing parameters"` to the attribute name instead?